wordpress-mobile/gutenberg-mobile
# Update release automation script to create WPiOS and WPAndroid PRs

Here are the changes made to the script

- Replaced "npx json" usage with "jq" for faster execution
- Added string formatting, execute and abort utils
- The script can now create WPiOS and WPAndroid PRs

To test:
1. Run the script and use an unusually high version number 
1. Check if Gutenberg, GB-Mobile, WPiOS and WPAndroid PRs are created as drafts
1. Check if PRs have correct descriptions and changes
1. Close the draft PRs

PR submission checklist:
